src/main/resources/assets/admin/common/js/app/view/ItemStatisticsHeader.ts

```diff
@@ -7,81 +7,74 @@ import {ImgEl} from '../../dom/ImgEl';
 import {ImgHelper} from '../../dom/ImgElHelper';
 import {SpanEl} from '../../dom/SpanEl';
 import {ViewItem} from './ViewItem';
+import * as Q from 'q';
 
 export class ItemStatisticsHeader
     extends DivEl {
 
-    private browseItem: ViewItem;
+    protected browseItem: ViewItem;
 
-    private iconEl: Element;
+    protected iconEl: Element;
 
-    private readonly headerTitleEl: H1El;
+    protected readonly headerTitleEl: H1El;
 
-    private readonly headerPathEl: H4El;
+    protected readonly headerPathEl: H4El;
+
+    protected readonly iconBlock: Element;
 
     constructor() {
         super('header');
+
+        this.iconBlock = new DivEl('icon-block');
         this.headerTitleEl = new H1El('title');
         this.headerPathEl = new H4El('path');
-        this.appendChild(this.headerTitleEl);
-        this.appendChild(this.headerPathEl);
     }
 
     setItem(item: ViewItem) {
-        if (this.iconEl) {
-            this.iconEl.remove();
+        this.browseItem = item;
+        this.iconEl?.remove();
+
+        if (!item) {
+            return;
         }
 
-        if (item) {
-            this.iconEl = this.createIconEl(item);
-            this.prependChild(this.iconEl);
+        this.iconEl = this.createIconEl().addClass('font-icon-default');
+        this.iconBlock.appendChild(this.iconEl);
 
-            let displayName = item.getDisplayName() || '';
-            this.headerTitleEl.setHtml(displayName);
-            this.headerTitleEl.getEl().setAttribute('title', displayName);
+        const displayName: string = item.getDisplayName() || '';
+        this.headerTitleEl.setHtml(displayName);
+        this.headerTitleEl.getEl().setAttribute('title', displayName);
 
-            this.headerPathEl.removeChildren();
-        }
-
-        this.browseItem = item;
+        this.headerPathEl.removeChildren();
     }
 
     setIconUrl(value: string) {
-        if (this.iconEl) {
-            this.iconEl.remove();
-        }
+        this.iconEl?.remove();
 
-        let size = this.getIconSize(this.browseItem);
-        let icon: HTMLImageElement = ImageLoader.get(value, size, size);
+        const size: number = this.getIconSize();
+        const icon: HTMLImageElement = ImageLoader.get(value, size, size);
 
-        this.iconEl = <ImgEl> new Element(new NewElementBuilder().setTagName('img').setHelper(
-            new ImgHelper(icon)));
-
-        this.prependChild(this.iconEl);
+        this.iconEl = <ImgEl> new Element(new NewElementBuilder().setTagName('img').setHelper(new ImgHelper(icon)));
```

Comment:
@ashklianko This can be extracted to a separate method:

``` Typescript
private doCreateIconEl(src: string): Element {
  const size: number = this.getIconSize();
  const icon: HTMLImageElement = ImageLoader.get(src, size, size);

  return new Element(new NewElementBuilder().setTagName('img').setHelper(new ImgHelper(icon)));
}

setIconUrl(value: string) {
  ...
  this.iconEl = this.doCreateIconEl(value);
  this.iconBlock.appendChild(this.iconEl);
}

private createIconEl() {
  ...
  if (this.browseItem.getIconUrl()) {
    return this.doCreateIconEl(this.browseItem.getIconUrl());
  }
  ...
}
```